model/project.go

```diff
@@ -473,6 +473,13 @@ func (m Module) GetOwnerAndRepo() (string, string, error) {
 	return m.Owner, m.Repo, nil
 }
 
+// CreateName creates a name based on owner, repo and branch
+func (m Module) CreateName() string {
```

Comment:
I think this isn't great for module/project.go because this struct already has a name field.